src/app/pages/debrief/components/safety-and-balance-card/safety-and-balance-card.cat-a-mod2.html

```diff
@@ -6,10 +6,10 @@
   </ion-card-header>
   <ion-card-content>
     <ion-grid>
-      <ion-row class="ion-align-items-center mes-data" *ngFor="let question of safetyAndBalanceQuestions$| async">
+      <ion-row class="ion-align-items-center row-padding mes-data" *ngFor="let question of safetyAndBalanceQuestions$| async">
         <ion-col size="32" class="ion-no-padding"></ion-col>
-        <ion-col size="3" class="down-padding ion-no-padding ion-align-self-start">
-          <div class="counter-icon">
+        <ion-col size="auto" class="ion-no-padding">
```

Comment:
auto sounds dangeroly like letting it decide if we want it by itself.  do we not want to control this?